pkg/store/proxy.go

```diff
@@ -242,13 +239,13 @@ func (s *ProxyStore) LabelValues(ctx context.Context, r *storepb.LabelValuesRequ
 	)
 	stores, err := s.stores(ctx)
 	if err != nil {
-		return nil, grpc.Errorf(codes.Unknown, err.Error())
+		return nil, status.Errorf(codes.Unknown, err.Error())
```

Comment:
I think we should be using `grpc.Errorf` consistently. I didn't know it existed initially that's why we have `status.` a lot. But `grpc` actually wraps `codes` and `status` and provides helper to extract the code from errors.

`status.*` also just seems odd because it's a pretty generic name even though it's grpc specific.

Reply:
wait. Isn't `grpc` one deprecated?

Reply:
Oh, you just meant the function. Checked and you are totally right. Interesting since `status` and `codes` always seemed a bit obfuscating to me. 

But fine, let's stick with the past approach then.

Reply:
past means what? `grpc` or `status` package? (: